src/screens/MessageScreen.tsx

```diff
@@ -286,6 +315,15 @@ const defaultStyles = createStyles('MessageScreen', (theme) => {
       color: theme.colors.text,
       fontWeight: '600',
     },
+    multiGiftedAvatarView: {
+      flex: 1,
+      flexWrap: 'wrap',
+      maxHeight: 40,
+      marginRight: 50,
+    },
+    profileView: {
+      height: 20,
+    },
```

Comment:
I think if we throw a justify center on here it would look a bit better when there are just three. I think it would look like this
![image](https://github.com/lifeomic/react-native-sdk/assets/2295908/cf3e9203-9c54-4194-92dd-88233ffadb9f)